src/views/Settings.vue

```diff
@@ -123,21 +125,23 @@ export default defineComponent({
       });
       return popover.present();
     },
+    setFacility (facility: any) {
```

Comment:
We should not change the position. Move it after logout and only specific changes should be visible

Reply:
Made the required changes.